_posts/2016-07-01-newsletter.md

```diff
@@ -0,0 +1,323 @@
+  Command-t and vim-github-dashboard).
+  - To enable Ruby support, just `gem install neovim`. You don't need to worry
+    about compiling against a specific version.
+- `:CheckHealth` is like homebrew "doctor", conceived by @tweekmonster and
+  extended by @tjdevries. We look forward to adding more and more "sanity
+  checks" covering common problems. In the future, instead of wondering whether
+  your runtime is broken or your `$TERM` is wrong (or ...), reach for
+  `:CheckHealth`. Users should not need to fumble with the problems that
+  experienced users just "learn to deal with".
+
+Project changes
+---------------
+
+> clarity and consistency breed contribution
+> — @robertmeta
+
+### Funding: Nikolai Pavlov (ZyX) assumes Thiago's role
+
+The enormously successful
+[salt campaign](https://salt.bountysource.com/teams/neovim) has funded
+Thiago's work, yielding libmpack, major refactors and improvements. For personal
+reasons Thiago will take a less active role in the project, and the 2-3 days of
+funded work will now be fulfilled by ZyX, the second-most [prolific](#note1)
+committer to Neovim _and Vim_ (~50,000 LOC contributed to Neovim, ~13,000 LOC
+contributed to Vim—including `if_python`, Vim's most mature FFI).
+
+ZyX's work on Neovim includes: first-class XDG support, shada, build-time
+generators, automated change-aware linter, lua-VimL. He gave careful
+attention to internals such as `msgpackparse()`/`msgpackdump()`, `string()`, and
+completely re-wrote `json_decode()`/`json_encode()` support (incl. granular
+error messages and comprehensive test coverage).
+
+
+Release strategy
+----------------
+
+### 0.1.x releases
+
+In November we announced our first release, `0.1.`. Since then we have
+streamlined our release process (versioning, tagging, changelogging,
+announcement-making, booty-shaking).
+
+I consider these releases reasonably stable, though we still have some bugs that
+we'd rather not have. With `0.2`, Windows will become a first-class target.
+
+Some question the lack of a `1.0` sticker. We follow
+[semver](https://semver.org), the recommendations there explain the status that
+we want to signal with the `0.x` version designations. We don't want to declare
+`1.0` until we have some features like providers, UIs, and the remote API
+designed to a spec that we want to commit to supporting for the (very) long
+term.
+
+### OS Packages
+
+More OS packages are appearing. Thanks to @jamessan, @fwalch and others for
+building packages for their favorite systems and working with us at
+neovim/neovim to address inevitable compiler/platform quirks that come from
+supporting dozens of targets. Neovim is on its way to
+[debian](https://packages.debian.org/experimental/neovim).
+
+Progress
+--------
+
+What did we _really_ do?
+
+50,000 (TODO: exact number?) new lines of code have been written. That's 25% of the codebase we
+started with.
+1000 (TODO:?) _new_ tests (on top of Vim's existing test suite).
```

Comment:
Since the beginning of the project? Or since 0.1.4? Or since the last newsletter?